CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.0
+
```

Comment:
I'm a little torn.

On the one hand, we don't have any public API changes for the component, but on the other, we're completely changing the component out and replacing the animation.

Reply:
looks like there are some who agree that color updates are breaking changes https://medium.com/eightshapes-llc/visual-breaking-change-in-design-systems-1e9109fac9c4 - in there the author explains that because other colors can easily be overridden or themed, changing one can break a UI.

Reply:
We'll review this at Groom and Prune tomorrow afternoon, FYI. I tend to lean towards breaking change for this but that's based on only a cursory review. 

And at the next ComponentDome let's talk more about Nathan's suggestion from that article: "Review the vast collection of possible CSS properties and discuss the consequences of candidate properties with your team. A working session will fruitfully reveal the group’s tolerance for protecting adopters and drive towards documenting how far you’ll go." Thx!

Reply:
After a conversation this week with some front-enders, it seems it's best to make this a breaking change.  Additionally, we have another breaking change coming summarily, so there's little reason not to just "do the right thing" here.